template-parts/home-sections/image_block.php

```diff
@@ -11,10 +11,9 @@
 }
 
 ?>
-
+<div class="img-container">
+<a href="<?php echo esc_attr( get_sub_field( 'image_link' ) ); ?>" class="image_link"?>
 <div class="background-image" style="background-image: url('<?php echo esc_attr( wp_get_attachment_image_url( get_sub_field( 'background_image'), 'full' ) ); ?>');">
 </div>
-
-<a href="<?php echo esc_attr( get_sub_field( 'button_url' ) ); ?>" class="button <?php echo get_sub_field( 'theme_color' ); ?>"<?php echo ( ! empty( get_sub_field( 'open_button_in_new_tab' ) ) ? ' target="_blank"' : '' ); ?>>
-	<?php echo get_sub_field( 'button_text' ); ?>
-</a>
\ No newline at end of file
+</a></a>
+</div>
```

Comment:
Please indent this code block appropriately. This file is indented by Tabs rather than Spaces, but in this case you included no indenting at all.

Reply:
This code block still needs indented. The `<a>` tag needs indented one level and the `<div>` inside the `<a>` tag needs indented two levels deep.